pysqldb3/data_io.py

```diff
@@ -421,9 +421,31 @@ def sql_to_sql(from_sql, to_sql, org_table, LDAP_from=False, LDAP_to=False, spat
     if not dest_table:
         dest_table = org_table
 
+    sql_to_sql_qry(from_sql, to_sql,
+           f'select * from [{org_schema}].[{org_table}]', LDAP_from, LDAP_to, spatial, org_schema, dest_schema, print_cmd, dest_table, temp,
+           gdal_data_loc, pg_encoding, permission)
+
+    cols = from_sql.get_table_columns(org_table, schema=org_schema)
+    cols = {c[0]:c[1] for c in cols}
+    cols_to = to_sql.get_table_columns(dest_table, schema=dest_schema)
+    cols_to = {c[0]: c[1] for c in cols_to}
+
+    # enforce column data types in dest match the source
+    for c in cols.keys():
+        if '-' in c:
+            # GDAL sanitizes `'` out of column names this will put them back so the start/end tables match
```

Comment:
for this comment, do you mean that "# GDAL sanitizes '-' out of column names " instead?